web-components/src/components/inputs/LocationStateField.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Field } from 'formik';
+import { makeStyles } from '@mui/styles';
+import { Theme } from '../../theme/muiTheme';
+
+import SelectTextField, { Option } from './SelectTextField';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  textField: {
+    '& .MuiInputBase-formControl': {
+      marginTop: theme.spacing(2.25),
+    },
+  },
```

Comment:
seems like this style is repeated in both `LocationStateField` and `LocationCountryField`
we'd rather pass this style as props from the upper component (retire form)